numpy/f2py/utils.py

```diff
@@ -0,0 +1,29 @@
+"""Global f2py utilities."""
+
+import os
+import contextlib
+import tempfile
+import pathlib
+import shutil
+
+def get_f2py_dir():
+	"""Return the directory where f2py is installed."""
+	return os.path.dirname(os.path.abspath(__file__))
```

Comment:
Could we use `pathlib`? e.g. as in `utils.py`

Reply:
Yes, made the changes, please have a look.